### PR TITLE
Basic Auth update

### PR DESCRIPTION
**Pull request by pbafna03**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: teams/teams-ps/teams/Connect-MicrosoftTeams.md [hunk 1]
```diff
@@ -44,8 +44,7 @@ Connect-MicrosoftTeams
 ## DESCRIPTION
 The Connect-MicrosoftTeams cmdlet connects to Microsoft Teams with an authenticated account for use with cmdlets from the MicrosoftTeams PowerShell module. After executing this cmdlet, you can disconnect from MicrosoftTeams account using Disconnect-MicrosoftTeams.
 
-> [!WARNING]
->If basic authentication is not enabled, legacy *-Cs cmdlets will not function properly. For Remote PowerShell, basic authentication is necessary.
+**Note**: With versions 4.x.x or later, enablement of basic authentication is not needed anymore in Commercial Environments. For GCC High / DoD environments & customers that are or have previously been enabled for Regionally Hosted Meetings for Skype for Business Online, basic authentication needs to be enabled for *-Cs cmdlets to function properly.
```
Comment (dariomws) [inline teams/teams-ps/teams/Connect-MicrosoftTeams.md:47]:
```suggestion
**Note**: With versions 4.x.x or later, enablement of basic authentication is not needed anymore in Commercial environments. For GCC High/DoD environments and customers that are, or have previously been enabled for Regionally Hosted Meetings in Skype for Business Online, basic authentication needs to be enabled for *-Cs cmdlets to function properly.
```

**Review by dariomws — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

File: teams/teams-ps/teams/Connect-MicrosoftTeams.md [hunk 2]
```diff
@@ -44,8 +44,7 @@ Connect-MicrosoftTeams
 ## DESCRIPTION
 The Connect-MicrosoftTeams cmdlet connects to Microsoft Teams with an authenticated account for use with cmdlets from the MicrosoftTeams PowerShell module. After executing this cmdlet, you can disconnect from MicrosoftTeams account using Disconnect-MicrosoftTeams.
 
-> [!WARNING]
->If basic authentication is not enabled, legacy *-Cs cmdlets will not function properly. For Remote PowerShell, basic authentication is necessary.
+**Note**: With versions 4.x.x or later, enablement of basic authentication is not needed anymore in Commercial environments. For GCC High/DoD environments and customers that are, or have previously been enabled for Regionally Hosted Meetings in Skype for Business Online, basic authentication needs to be enabled for *-Cs cmdlets to function properly.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Connect-MicrosoftTeams.md:47]:
```suggestion
**Note**: With versions 4.x.x or later, enablement of basic authentication is not needed anymore in commercial environments. For GCC High/DoD environments and customers that are, or have previously been enabled for Regionally Hosted Meetings in Skype for Business Online, basic authentication needs to be enabled for *-Cs cmdlets to function properly.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @pbafna03, kindly commit the minor suggested change. Many thanks.

**Comment by pbafna03** [comment]
@JohanFreelancer9  - Done!

**Review by tseward — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**